### DISCUSSION TITLE
Does it have a Ticker component?

### DISCUSSION THREAD

**Discussion by mjr** [motiondivision/motion#2203] [Q&A]
I didn't understand very well, I saw in this article (https://www.framer.com/learn/create-carousels-tickers-slideshows/) about the Carousel and the Ticker component powered by Framer Motion, but I can't find anywhere about how to use it. Is this specific to Framer and doesn't have an implementation in React or does it exist somewhere?

**Comment by Anil-07** [comment]
@mjr Hi, Iam trying to implement vertical ticker , using framer motion in my reactjs project ,found any solution ?
